sentences/nl/fan_HassTurnOn.yaml

```diff
@@ -2,7 +2,15 @@ language: nl
 intents:
   HassTurnOn:
     data:
-    - sentences: []
-      slots:
-        domain: fan
-        name: all
+      - sentences:
+          - "<doe> [alle] [de] (ventilator | ventilatoren) in <area> [<naar>] aan"
```

Comment:
[(alle | de)]

Reply:
Wat doet “naar” hier?

Reply:
"naar" zit er in voor mensen die "schakel de ventilator in de woonkamer naar aan" zeggen. Ik zou het zelf niet snel doen, maar ik vind het ook niet helemaal uitgesloten.